gui/src/components/WorkspaceList.svelte

```diff
@@ -5,6 +5,10 @@
   export let workspaces: WorkspaceDescription[];
 </script>
 
+<button on:click={() => router.push('#/new-workspace')}>
```

Comment:
We really need a `Button` component that supports `href` to specify this standard on-click behavior.